docker-compose.yml

```diff
@@ -1,6 +1,19 @@
 version: '3'
 
 services:
+  frontend:
+    build: ./frontend
+    volumes:
+      - ./frontend:/app
+      # One-way volume to use node_modules from inside image
+      - /app/frontend/node_modules
+    ports:
+      - "3000:3000"
+    environment:
+      - NODE_ENV=development
+    depends_on:
+      - django-app
+    command: npm start
```

Comment:
Can we put it as `CMD ["npm", "start"]` in Dockerfile?
It's better to not overwhelm the compose file with details.